### ISSUE TITLE
make small video for ?-Popup

### ISSUE THREAD

**Issue by star-buck** [xdgurl/xdgurl#7]
add small introduction video in ?-Popup, then we can make "install" buttons visible for everyone on thursday.

**Comment by star-buck** [comment]
add small introduction video in ?-Popup, then we can make "install" buttons visible for everyone on thursday.

**Comment by star-buck** [comment]
@akiraohgaki : Any progress with that?
Can we also switch again to iframe for the popup, then you me know how one can edit the "Instructins" website?

**Comment by akiraohgaki** [comment]
I just editing the video, please wait for few hours.

**Comment by star-buck** [comment]
then lets prepare the github xdgurl webpage + iframe again:
@akiraohgaki : Can you email me the link to installation instructions and access to edit?

**Comment by star-buck** [comment]
You can also add youtube video URL in your product and it comes up in a tab called "media":
https://www.pling.com/p/1136805/

**Comment by akiraohgaki** [comment]
OK, here is the video. What do you think?

https://youtu.be/DXHEHo1oKrw

**Comment by akiraohgaki** [comment]
Installation instructions from xdgurl.github.io is back.

**Comment by star-buck** [comment]
video is nice, a bit long though.

I also miss clear installation instructions after 30 seconds unity demo, for example like
1. go to linux-apps.com/url
2. click on .deb to bring up qapt, then install with dependencies.
3. click on install button, associate url for firefox with xdgurl (how does chrome work?)
4. popup comes up, click install
5. go to settings of your DE, find theme and "Apply"

then after that, how about adding more TEXT overlay for the following parts, such as 
xdgurl on xfce, xdgurl on plasma, etc.?

**Comment by akiraohgaki** [comment]
OK, I will make the video, again.

**Comment by star-buck** [comment]
in the meantime , we will switch ?-popup to external site again and update in 3-4 hours live

**Comment by star-buck** [comment]
also, its too fast cuts, its very hard to follow with your eyes the mouse cursor, plus the fast music is adding to this confusion...

**Comment by star-buck** [comment]
so just relax everything, make the cuts slower and give the user time to follow whats going on :)

**Comment by akiraohgaki** [comment]
Here is new video:
https://youtu.be/GuB-NRtahoA

This is very simple video, but little bit long.

Better than previous video?

**Comment by star-buck** [comment]
good tutorial video.
small typo: it is called "Tweaks"
we could link both videos then in iframe popup, install instructions at top, then showcase video at bottom.

**Comment by akiraohgaki** [comment]
I will fix text.
also make showcase video better.

2016年7月11日月曜日、star-buck<notifications@github.com>さんは書きました:

> good tutorial video.
> small typo: it is called "Tweaks"
> we could link both videos then in iframe popup, install instructions at
> top, then showcase video at bottom.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/xdgurl/xdgurl/issues/7#issuecomment-231728603>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/ABbxlgUk50dpH9nqAVUkj1gkBZnaGfR_ks5qUj_4gaJpZM4JFKtg>
> .
>


-- 

Regards,

Akira Ohgaki <akiraohgaki@gmail.com>

**Comment by star-buck** [comment]
imo you could also combine the two videos, with 45-60 seconds showcase, then do install instructions.

also you have a nice credits at the end, you could show XDGURL logo with Text "Introduction" first 5 seconds, then show parts of first video, then add "Installation" part from second video, with credits from first video at end.

**Comment by star-buck** [comment]
so you have a 3 minutes total video with nice xdgurl logo at start, and nice credits at end.

**Comment by akiraohgaki** [comment]
New video here:
https://youtu.be/MtW-4mAXUwI

Maybe this is better than previous videos.

**Comment by akiraohgaki** [comment]
And I would like to remove previous videos later.

**Comment by star-buck** [comment]
The flow and style of new video is great.

Now the music is too apple/common these days imo, do you have a site with
free music to try?

Also why not show this under Unity or XFCE?
Showing KDE will kinda have the people reminded that there is already GHNS,
which the others dont.

Just saying, XFCE or Gnome would be auite impressive as they lack direct
integration.

So instead of randomly showing various iconthemes a d windows decorations
at the beginning, why not show quickly a 3-4 examples:
1. Website product + files tab + install button click
2. settings place of DE + apply
3. Result

15 seconds total per case

1. Gnomeshell under gnome
2. Window deco under xfce
3. Plasma lnf theme
4. Icon theme under Enlightenment or other theme

Text overlay "gnome-look.org" or "Gnome", etc.

Just thinking to squeeze the most impressive video for users out there,like
a commercial...

So style and flow are good, but information could be much more interesting
for viewer... seeing icons and different colored shells is a bit boring
without getting shown the magic behind xdgurl 4xtimes repeated to make
"click" with the viewer and make him want to know how to install it
watching the second half...

**Comment by akiraohgaki** [comment]
hmm... qapt(so KDE), combine showcase + tutorial.
It's not good idea, as already we know...

I will make installation instructions video as opendesktop.org's help content.

**Comment by akiraohgaki** [comment]
New video:
https://youtu.be/l3vXR0uh7zk

Can we replace current video to this?
